bciers/package.json

```diff
@@ -4,7 +4,9 @@
   "license": "MIT",
   "scripts": {
     "dev": "nx dev reporting",
-    "reg": "nx dev registration"
+    "reg": "nx dev registration",
+    "reg:coverage": "nx run registration:test --coverage",
+    "reg:test": "nx run registration:test"
```

Comment:
hot reloading is currently not working. To enable it I believe we will have to update `nx`, so I decided to make it's own issue: 

https://github.com/bcgov/cas-registration/issues/1610